rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/RxBleConnectionImpl.java

```diff
@@ -528,4 +533,32 @@ public BluetoothGattCharacteristic call(byte[] bytes) {
     public Observable<Integer> readRssi() {
         return rxBleRadio.queue(new RxBleRadioOperationReadRssi(gattCallback, bluetoothGatt, timeoutScheduler));
     }
+
+    @Override
+    public <T> Observable<T> queue(final RxBleRadioOperationCustom<T> operation) {
+        return rxBleRadio.queue(new RxBleRadioOperation<T>() {
```

Comment:
This would be wrong IMO, I'm flashing a firmware, I don't want to have a maximum amount of time to perform the operation as the default timeout could not fit what I need to do. 

We would need a rather larger timeout which would defeat the purpose.